.github/workflows/import_count.yml

```
@@ -0,0 +1,57 @@
+name: Compute transitive imports
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        sudo apt-get install -y jq
+        # If you have additional dependencies, install them here
+
+    - name: Compute transitive imports
+      run: |
+        # Compute the counts for the HEAD of the PR
+        python ./scripts/count-trans-deps.py > head.json
+
+        # Checkout the merge base
+        git checkout $(git merge-base refs/remotes/origin/main ${{ github.sha }})
```

```suggestion
        git checkout "$(git merge-base refs/remotes/origin/main ${{ github.sha }})"
```
I think that this is what shellcheck is saying.